cetloxblock/cetloxblock.pyproj.vspscc

```diff
@@ -0,0 +1,10 @@
+﻿""

```

Comment:
When you make a `.gitignore` file, this should also be ignored if it isn't already, i.e. `*.vspscc` files.